distributed/tests/test_client.py

```diff
@@ -1116,6 +1117,27 @@ def g():
     result = yield y._result()
     assert result == 456
 
+@gen_cluster(client=True)
+def test_upload_file_zip(c, s, a, b):
```

Comment:
I'm not sure to get this one.
This module is uploaded locally on workers , right ? The workers aren't restarted before each test ?